platform/osx/bitrise.yml

```diff
@@ -33,6 +33,8 @@ workflows:
             export BUILDTYPE=Debug
             make osx
             make test-osx
+            make linux
+            make test-linux
```

Comment:
From https://www.bitrise.io/build/80b3fbddc8ce5bda, it looks like core code would indeed get rebuilt from scratch. Generally, builds of master have been taking 5–10 minutes, but we expect the build time to go up if we decide to run iOS SDK UI tests on Bitrise: #4837.